apps/i18n/common/en_us.json

```diff
@@ -985,6 +985,9 @@
   "downloadParentLetter": "Download parent letter",
   "downloadReplayVideoButtonDownload": "Animation",
   "downloadReplayVideoButtonError": "Sorry, we were unable to download your animation. Please try re-running your project and trying again.",
+  "downloadUnitLessonPlans": "Download all lesson plans for Unit {unitNumber}",
+  "downloadUnitSlides": "Download all slides for Unit {unitNumber}",
```

Comment:
Nit: should we remove the slides until we add it later, or is it worthwhile to do now?